holidata/holidays/es-ES.py

```diff
@@ -3,22 +3,58 @@
 
 from .holidays import Locale
 
+"""
+Information taken from government websites around 2020-06
+    https://administracion.gob.es/pag_Home/atencionCiudadana/calendarios/laboral.html
+    http://www.seg-social.es/wps/portal/wss/internet/CalendarioLaboral
+    2020: https://www.boe.es/diario_boe/txt.php?id=BOE-A-2019-14552
+    2019: https://www.boe.es/diario_boe/txt.php?id=BOE-A-2018-14369
+
+Regional governments
+    [AN] https://www.juntadeandalucia.es/temas/trabajar/relaciones/calendario.html
+
+Also those sites for some information
+    https://es.wikipedia.org/wiki/Calendario_laboral
+"""
 
 class es_ES(Locale):
     """
     01-01: [NF] Año Nuevo
-    01-06: [NRF] Día de los Reyes
+    01-06: [NRF] Epifanía del Señor
+    02-28: [AN] [F] Día de Andalucía
+    03-01: [IB] [F] Día de las Illes Balears
+    03-13: [ML] [F] Estatuto de Autonomía de la Ciudad de Melilla
+    03-19: [GA,MC,NC,PV,VC] [RF] San José
+    04-23: [AR] [RF] San Jorge / Día de Aragón
```

Comment:
This is the same as for Galicia national day. This is a holiday with a religious origin. It is the day of _San Jorge_ and due to the relation of this region with this saint it is picked as region's day. Other regions only have _Día de_ because they not have a religion origin or they have just decided to drop its religious origin link. This could be a local decision.

As you can read in the first sentence of the above [Spanish wikipedia link](https://es.wikipedia.org/wiki/D%C3%ADa_de_Arag%C3%B3n):
> Oficialmente conocido como El Día de Aragón, se celebra el 23 de abril para homenajear a San Jorge como santo patrón del Reino de Aragón.

Reply:
I also prefer simpler solution. But I really don't know what to do with that. Official government documentation refer to this day as _San Jorge / Día de Aragón_.

I do not have lived in any of those regions (_Galicia_ or _Aragón_) so I can not tell if we can drop the "Saint" part. As I said before, it is well know the importance of Santiago (Saint Jacques) for _Galicia_ but I don't know if the same is true for _Aragón_ with Saint George.